import.py

```diff
@@ -644,6 +656,27 @@ def compile_base(compile_script: str, source: str, c_file: str, out_file: str) -
         print("Warning: failed to compile .c file.")
 
 
+def create_write_settings_toml(
+    func_name: str, compiler_type: str, filename: str
+) -> None:
+
+    all_weights = load_weights_from_file()
+    base_weights = all_weights["base"]
+    compiler_weights = all_weights[compiler_type]
+
+    with open(filename, "w", encoding="utf-8") as f:
+        f.write(f'func_name = "{func_name}"\n')
+        f.write(f'compiler_type = "{compiler_type}"\n\n')
+
+        f.write("# uncomment lines below to customize the weights\n")
+        f.write("# run --help=randomization to see doc\n")
+        f.write("[custom_weights]\n")
```

Comment:
(This name bugs me a little: it's kinda implicit that these are custom since we're in a file specifically meant to customize things. Maybe change to something like `[randomization_weights]` or `[perm_weights]` or `[weight_overrides]`? May also be fine to keep though.)

Reply:
I like weight_overrides